docs/sources/datasources/prometheus/query-editor/index.md

```diff
@@ -68,43 +72,45 @@ You can switch between **Table**, **Time series**, and **Heatmap** options by co
 | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **Table**       | This works only in a [Table panel]({{< relref "../../../panels-visualizations/visualizations/table" >}}).                                                                                                                           |
 | **Time series** | Uses the default time series format.                                                                                                                                                                                                |
+| **Table**       | This works only in a [Table panel]({{< relref "../../../panels-visualizations/visualizations/table" >}}).                                                                                                                           |
 | **Heatmap**     | Displays metrics of the Histogram type on a [Heatmap panel]({{< relref "../../../panels-visualizations/visualizations/heatmap" >}}) by converting cumulative histograms to regular ones and sorting the series by the bucket bound. |
 
 ### Type
 
-The **Type** setting selects the query type.
-
-- A **Range** query returns a Range vector, comprised of a set of time series containing a range of data points over time for each time series.
-- An **Instant** query returns only the latest value that Prometheus has scraped for the requested time series. Instant queries can return results much faster than normal range queries and are well suited to look up label sets.
-
-  Instant query results are comprised of only one data point per series and can be shown in the time series panel by adding a field override, adding a property to the override named `Transform`, and selecting `Constant` from the **Transform** dropdown.
-
-  For more information, refer to the [Time Series Transform option documentation]({{< relref "../../../panels-visualizations/visualizations/time-series#transform" >}}).
+The **Type** setting sets the query type.
 
+- The **Both** query option is the default option and returns results for both a **Range** query and an **Instant** query.
+- A **Range** query returns a range vector consisting of a set of time series data containing a range of data points over time for each time series. You can choose lines, bars, points, stacked lines or stacked bars
+- An **Instant** query returns one data point per query and only the most recent point in the time range provided. Instant query results can be depicted in the time series panel by adding a field override, adding a property to the override named `Transform`, and selecting `Constant` from the **Transform** dropdown. The results can be shown in table format or as raw data.
```

Comment:
Suggestion here, re order the instant query language. I think I misunderstood the purpose of the instructions around adding an override. If a user does an instant query, the default vis is table or raw (these views do not require anything to show the query). But if a user wants to display their instant query in the vis along with other time series that are ranges, they do this by following the instructions here. Let me know if we need more clear instructions for this part!

An Instant query returns one data point per query and only the most recent point in the time range provided. The results can be shown in table format or as raw data. To depict instant query results in the time series panel, first add a field override, next add a property to the override named Transform, and finally select Constant from the Transform dropdown.

Reply:
changed per suggestion